EIPS/eip-5850.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 5850
+title: Complex Numbers stored in Bytes32 types
+description: Store real and imaginary parts of complex numbers in the lower 16 bytes and upper 16 bytes respectively of a bytes32 type.
+author: Paul Edge (@genkifs)
+discussions-to: https://ethereum-magicians.org/t/TBC
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-10-29
+---
+ 
+## Abstract
+
+This EIP proposes a natural way for complex numbers to be stored in the bytes32 data-type.
+
+## Motivation
+
+This EIP proposes to package both the real and imaginary within one existing data type, Bytes32.  This allows compact storage without the need for structures and facilitates easy library implementations.  The bytes32 would remain available for existing, non-complex uses.
+
+## Specification
+
+A complex number would be defined as Bytes32 and initalized with the `cnNew` function and converted back with `RealIm`, both given below.
+
+## Rationale
+
+Only the split is defined in this EIP.  Manipulation of complex numbers (addition, multiplication etc.), number of decimal places and other such topics are left for other EIP discussions.
```

Comment:
Why?

Reply:
To keep the EIP focused on a single simple idea.  Manipulation of complex numbers when they are in Byte32 form is undoubtedly a *much* larger topic.